res/database/Default/database_ads.xml

```diff
@@ -230,19 +231,19 @@ You will really have to chew.</en>
 			</title>
 			<description>
 				<de>Action-Spielfiguren Deiner Lieblingspolitiker. Für kurze Zeit Action-Märkel und Chaoz-Rözler zum Preis von einem.</de>
-				<en>Action game figures of your favorite politician. For short time Action-Murkel and Chaoz-Ruszler for the price of one.</en>
+				<en>Action play figures of your favorite politician. For short time Action-Murkel and Chaoz-Röszler for the price of one.</en>
```

Comment:
"Ruszler" is more distinct from the original name than "Röszler".

Reply:
I didn't change Murkel. And I thought "Ruszler" would make him too unrecognizable.